modules/monitoring-attaching-additional-labels-to-your-time-series.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * monitoring/cluster_monitoring/configuring-the-monitoring-stack.adoc
+
+[id="attaching-additional-labels-to-your-time-series_{context}"]
+= Attaching additional labels to your time series
+
+Using the external labels feature of Prometheus, you can attach additional custom labels to all time series and alerts leaving the Prometheus cluster.
+
+.Prerequisites
+
+* Make sure you have the `cluster-monitoring-config` ConfigMap object with the `data/config.yaml` section.
```

Comment:
It's not clear to me what to do here. Maybe have a one-liner command to create an empty ConfigMap?

```
oc create -n openshift-monitoring configmap cluster-monitoring-config --from-literal 'config.yml={}'
```

Reply:
@simonpasquier This is the convention we use in every section that involves editing `cluster-monitoring-config`, beginning with [1] . Ideally, we would link to [2], which answers your question, but we have to resort to just referring to it with this prerequisite line without linking.
[1] https://docs.openshift.com/container-platform/4.4/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#configuring-the-cluster-monitoring-stack_configuring-monitoring
[2] https://docs.openshift.com/container-platform/4.4/monitoring/cluster_monitoring/configuring-the-monitoring-stack.html#creating-cluster-monitoring-configmap_configuring-monitoring